WcaOnRails/db/seeds/round_types.seeds.rb

```diff
@@ -6,10 +6,10 @@
 ('2', 50, 'Second round', 'Second', 0),
 ('3', 79, 'Semi Final', 'Semi Final', 0),
 ('b', 39, 'B Final', 'B Final', 0),
-('c', 90, 'Combined Final', 'Combined Final', 1),
-('d', 20, 'Combined First round', 'Combined First', 0),
-('e', 59, 'Combined Second round', 'Combined Second', 0),
+('c', 90, 'Cutoff Final', 'Cutoff Final', 1),
+('d', 20, 'Cutoff First round', 'Cutoff First', 0),
+('e', 59, 'Cutoff Second round', 'Cutoff Second', 0),
 ('f', 99, 'Final', 'Final', 1),
-('g', 70, 'Combined Third round', 'Combined Third', 0),
+('g', 70, 'Cutoff Third round', 'Cutoff Third', 0),
 ('h', 10, 'Combined qualification', 'Combined qualification', 0);"
 ActiveRecord::Base.connection.execute(sql)
```

Comment:
In order for that to take effect you'd need to create a migration similar to [this one](https://github.com/thewca/worldcubeassociation.org/blob/d6bb76a8b8a70e7aaf1f38e1485ad9281e442684/WcaOnRails/db/migrate/20200125180554_update_feet_rank.rb).
(the Rails part of the website uses i18n phrases, so it doesn't really matter, some PHP code may still use these names though).